### DISCUSSION TITLE
Add support for Gpodder client

### DISCUSSION THREAD

**Discussion by hydrian** [thrillfall/nextcloud-gpodder#134] [Ideas]
Looking for a linux platform podcast player that supports this API. gpodder seems the obvious choice since it obviously supports the API. 

The support may even already exist, user just need documentation on how to set it up.

**Comment by thrillfall** [comment]
You need to ask this at https://github.com/gpodder/gpodder.

**Comment by ernestask** [comment]
https://github.com/gpodder/gpodder/issues/1251#issuecomment-1082264378

**Comment by thrillfall** [comment]
Any contributions that make this app more useful (to more users) are always welcome.

**Comment by hydrian** [comment]
I am a php developer and willing to help extend this app so the API so gpodder is usable but i have no experience in thr nextcloud app framework.

**Reply by thrillfall** [reply]
I was in exactly the same situation when I started this app. You now have the advantage of a working code base to build upon. 
Please go ahead. I am sure there are mutiple devs out there who will support when you might hit an obstacle!
